se.redfield.bert/src/se/redfield/bert/setting/BertClassifierSettings.java

```diff
@@ -186,8 +186,8 @@ public void loadSettingsFrom(NodeSettingsRO settings) throws InvalidSettingsExce
 		validationBatchSize.loadSettingsFrom(settings);
 		fineTuneBert.loadSettingsFrom(settings);
 		optimizer.loadSettingsFrom(settings);
-		classSeparator.loadSettingsFrom(settings);
 		multilabelClassification.loadSettingsFrom(settings);
+		classSeparator.loadSettingsFrom(settings);
```

Comment:
What is the motivation behind this edit? multilabelClassification has the listener attached to it that changes enabled status of the classSeparator. If the classSeparator is loaded last then it's enabled status is going to be overwritten from the settings.

Reply:
It's a bit counter-intuitive but that the loading overwrites the enabled status is actually correct behavior. Otherwise the classSeparator textfield is actually disabled if you load a workflow where multilabelClassification is enabled (e.g. https://kni.me/w/9nIhu3wrYPjBiVIf). That's because loading classSeparator will load the enabled status without notifying the DialogComponent. Then loading multilabelClassification triggers the listener which sets enabled to true but because it is already true, the DialogComponent of classSeparator is again not notified.
Arguably that's really a bug in KNIME AP but this workaround does ensure correct behavior for now.